### PR TITLE
style changes to text field in RJSF

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chatterjee <speak2adi@gmail.com>

**Description**

1. Added tooltip to text field. 

https://user-images.githubusercontent.com/53532328/149239499-29d1cf01-d97f-400d-b370-b791a7cf9a01.mov

- [x] still need to align the helpoutlineicon to the right

2. Changed the format of the tooltip for better readability. 

tooltip

can revert this change if it's not required. 
 

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by warunicorn19** [comment]
@leecalcote said that behaviour is looking neat 😅, I'll send a video with keeping the position fixed then we can decide which one looks better.

**Comment by warunicorn19** [comment]
@Chadha93 it turns out that we can't place an icon to the end of a mui textfield. Or at least I haven't been able to find it. @leecalcote is the above change fine? or do we want the icon to be fixed?

File: ui/components/MesheryMeshInterface/PatternService/EnlargedTextTooltip.js [hunk 1]
```diff
@@ -3,10 +3,13 @@ import { Tooltip } from "@material-ui/core";
 
 const EnlargedTextTooltip = withStyles((theme) => ({
   tooltip : {
-    backgroundColor : '#f5f5f9',
-    color : 'rgba(0, 0, 0, 0.87)',
-    fontSize : theme.typography.pxToRem(15),
+    backgroundColor : ' #fcfcfc',
+    color : '#1E2117',
```
Comment (Chadha93) [inline ui/components/MesheryMeshInterface/PatternService/EnlargedTextTooltip.js:7]: A defined color from figma?

File: ui/components/MesheryMeshInterface/PatternService/EnlargedTextTooltip.js [hunk 2]
```diff
@@ -3,10 +3,13 @@ import { Tooltip } from "@material-ui/core";
...
+    fontSize : theme.typography.pxToRem(12),
     border : '1px solid #dadde9',
+    borderRadius : "15px",
+    padding : "10px",
+
```
Comment (Chadha93) [inline ui/components/MesheryMeshInterface/PatternService/EnlargedTextTooltip.js:12]: Let's have these in rem

**Review by Chadha93 — CHANGES_REQUESTED** [review]
The video looks great  @warunicorn19 these are the desired changes as of now for the text fields.

Some minor comments to be addressed.

**Comment by warunicorn19** [comment]
Okay, pushing above changes.

**Comment by leecalcote** [comment]
> @Chadha93 @leecalcote thoughts on this?

It's beautiful. This another good step forward. As a future iteration, we might tighten margins, go two-column with the inputs, reduce font size on input fields. Thumbs-up on this PR.

File: ui/components/MesheryMeshInterface/PatternService/EnlargedTextTooltip.js (hunk 1 above)
Comment (leecalcote) [reply ui/components/MesheryMeshInterface/PatternService/EnlargedTextTooltip.js:7]: Right, yes. Good to check, though.

File: ui/components/MesheryMeshInterface/PatternService/EnlargedTextTooltip.js (hunk 2 above)
Comment (leecalcote) [reply ui/components/MesheryMeshInterface/PatternService/EnlargedTextTooltip.js:12]: Nice.

**Comment by Abhishek-kumar09** [comment]
Extra Whitespace on the right hand side of the forms, Are we planning something for it?

**Comment by Chadha93** [comment]
We could have two items in a row to eliminate right-side white space...

**Comment by leecalcote** [comment]
@Chadha93, this aligns with my suggestion. I'm not sure if it'll be too compact, but its worth trying.

**Pull request merged**